client/app/api/login/route.ts

```diff
@@ -0,0 +1,47 @@
+import { NextResponse } from "next/server";
+import { generateRandomToken, parseEmail } from "@/utils/helpers";
```

Comment:
nit: unused `generateRandomToken`

Reply:
👍 nits accepted :)

got to check my ESLint!